lib/events/filesessions/fileasync.go

```diff
@@ -389,6 +389,9 @@ func (u *Uploader) startUpload(ctx context.Context, fileName string) error {
 	if err != nil {
 		return trace.Wrap(err)
 	}
+
+	log := u.log.WithField("session-id", sessionID)
```

Comment:
```suggestion
	log := u.log.WithField("session", sessionID)
```

For consistency:

```
teleport/lib/events [$]
❯ ag 'WithField\("s'
stream.go
671:					WithField("session", w.proto.cfg.Upload.SessionID).
690:				WithField("session", w.proto.cfg.Upload.SessionID).

complete.go
181:		log := u.log.WithField("upload", upload.ID).WithField("session", upload.SessionID)
```

Reply:
I think we also use sid in other places (audit log), I kept it here as `session-id` for backwards compatibility with the existing debug log message. But I guess it's not a big deal and I can just change it to session.

Reply:
FWIW, there's also:

```
lib/events/s3sessions/s3stream.go
67:     h.WithFields(logrus.Fields{
68-             "upload":  aws.StringValue(resp.UploadId),
69-             "session": sessionID,
--
86:     log := h.WithFields(logrus.Fields{
87-             "upload":  upload.ID,
88-             "session": upload.SessionID,
--
113:    log := h.WithFields(logrus.Fields{
114-            "upload":  upload.ID,
115-            "session": upload.SessionID,
--
149:    log := h.WithFields(logrus.Fields{
150-            "upload":  upload.ID,
151-            "session": upload.SessionID,
--
188:    log := h.WithFields(logrus.Fields{
189-            "upload":  upload.ID,
190-            "session": upload.SessionID,

(azsessions.fieldSessionID is "session_id")
lib/events/azsessions/azsessions.go
345:    log := h.log.WithFields(logrus.Fields{
346-            fieldSessionID: upload.SessionID,
--
455:    h.log.WithFields(logrus.Fields{
456-            fieldSessionID:  upload.SessionID,

lib/web/sessions.go
138:            c.Log = log.WithFields(logrus.Fields{
139-                    "user":    c.User,
140-                    "session": c.Session.GetShortName(),
--
994:            Log: s.log.WithFields(logrus.Fields{
995-                    "user":    session.GetUser(),
996-                    "session": session.GetShortName(),

lib/web/terminal.go
124:                    log: logrus.WithFields(logrus.Fields{
125-                            trace.Component: teleport.ComponentWebsocket,
126-                            "session_id":    cfg.SessionData.ID.String(),

lib/web/command.go
457:                    log: logrus.WithFields(logrus.Fields{
458-                            trace.Component: teleport.ComponentWebsocket,
459-                            "session_id":    cfg.SessionData.ID.String(),
```